editor/src/clj/editor/buffers.clj

```diff
@@ -13,8 +13,9 @@
 ;; specific language governing permissions and limitations under the License.
 
 (ns editor.buffers
-  (:import [java.nio Buffer ByteBuffer ByteOrder IntBuffer]
-           [com.google.protobuf ByteString]))
+  (:require [util.num :as num])
+  (:import [com.google.protobuf ByteString]
+           [java.nio ByteBuffer ByteOrder DoubleBuffer FloatBuffer IntBuffer LongBuffer ShortBuffer]))
```

Comment:
* Fixed the low-level buffer population functions behavior with signed values and moved them to `buffers.clj`.